Java/README.md

```diff
@@ -39,6 +39,7 @@ _add list here_
 - [Longest Common Prefix](cp/Longest_Common_Prefix.java)
 - [Target Sum Triplets](cp/target_sum_triplets.java)
 - [Check for Subsequence](cp/CheckForSubSequence.java)
+- [Validate Stack Sequences](ValidateStackSequences.java
```

Comment:
@raksha009 Update like this `- [Validate Stack Sequences](cp/ValidateStackSequences.java)`